features/facebookImport/test/name-importer.test.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+import { ZipFileMock } from "./mocks/zipfile-mock";
+import { runNameImporter } from "./utils/data-importing";
+import {
+    expectImportSuccess,
+    expectInvalidContentError,
+    expectMissingFileError,
+} from "./utils/importer-assertions.js";
+
+const profileInformationFileName =
+    "profile_information/profile_information.json";
+
+function createProfileNameData(firstName, middleName, lastName) {
+    return {
+        profile_v2: {
+            name: {
+                full_name: [firstName, middleName, lastName]
+                    .filter((each) => each.length !== 0)
+                    .join(" "),
+                first_name: firstName,
+                middle_name: middleName,
+                last_name: lastName,
+            },
+        },
+    };
+}
+
+test("Name importer - missing file", async () => {
+    const zipFile = new ZipFileMock();
+    const { result } = await runNameImporter(zipFile);
+
+    expectMissingFileError(result);
+});
+
+test("Name importer - wrong data key", async () => {
+    const zipFile = new ZipFileMock();
```

Comment:
If you need some fixups, you can put them in a `beforeAll` function that's called before any test is invoked.

Reply:
I moved the creation of the mocked ZipFile to `beforeEach`